src/clj_test_containers/core.clj

```diff
@@ -179,13 +183,26 @@
      :stdout (.getStdout result)
      :stderr (.getStderr result)}))
 
+(defn dump-logs
+  [container-config]
+  ((:logs container-config)))
```

Comment:
My understanding is, that every time this function is called, the next log entries are streamed from the ToStringConsumer, right? I am wondering if we could return the logs as a lazy sequence, here, instead of calling the method multiple times. What do you think=

Reply:
Yes, you are right in your understanding. It could be a lazy seq I suppose too. Any idea how that might work as an example?

Reply:
I misread the initial testcontainers implementation, here. The logs are pushed to the consumer, so it would not make sense to access them lazily. But I was wondering if "get all the logs until now" is useful, or if there was a way to receive just a delta somehow, so developers can wait for a certain log event more easily

Reply:
That seems a little overly complicated for this first pass. A simple dump would suffice for now?

Reply:
Yes, I agree with you. The string log access will be useful to a lot of people already. I suggest changing the keywords `:capture-logs?` and `:logs` to be less generic, so they leave room to decide for different log strategies, later, e.g. using `Slf4JLogConsumer.java` or `WaitingConsumer.java` implementations by the testcontainers project. What do you think of `:string-logs` for a keyword in the results, and something similar to what you did for the Wait Strategies? I think that approach works really well for choosing a certain wait strategy and it might also work well, here.

Reply:
Sure, on it.

Reply:
Done and updated.

btw, you'll notice that in project.clj that I've commented out the githooks thing. It's conflicting with existing githooks that I have, and after reading the documentation https://github.com/gmorpheme/lein-githooks, it says it doesn't play nicely with existing setups. What are your thoughts on this one?